google/pubsub_v1/types/pubsub.py

```diff
@@ -1114,11 +1114,14 @@ class SubscriptionProperties(proto.Message):
         r"""Subscription properties sent as part of the response.
 
         Attributes:
+            exactly_once_delivery_enabled (bool):
+
             message_ordering_enabled (bool):
                 True iff message ordering is enabled for this
                 subscription.
         """
 
+        exactly_once_delivery_enabled = proto.Field( proto.BOOL, number=1,)
```

Comment:
Related to the first comment in the other review, should we persist this change through `owlbot.py` or are we expecting the proto change to go public before we merge this?

Reply:
Same answer as another. We expect the proto change to precede this change.